### PR TITLE
[api] api-platform.com

### PR DESCRIPTION
**Pull request by dupuchba**
Add API-Platform: a simple way to generate high quality APIs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -593,6 +593,7 @@ Libraries to help manage database schemas and migrations.
 * [Drest](https://github.com/leedavis81/drest) - A library for exposing Doctrine entities as REST resource endpoints.
 * [Restler](https://github.com/Luracast/Restler) - A lightweight framework to expose PHP methods as RESTful web API.
 * [wsdl2phpgenerator](https://github.com/wsdl2phpgenerator/wsdl2phpgenerator) - A tool to generate PHP classes from SOAP WSDL files.
+* [API PLATFORM](https://github.com/dunglas/api-platform) - Expose in minutes an hypermedia REST API that works out of the box that embraces JSON-LD, Hydra and provides a ton of features (also usable as standalone symphony bundle).
```
Comment (dunglas) [inline README.md]: API Platform (case)
https://api-platform.com (a dedicated website is in progress)
Symfony

**Comment by dupuchba** [comment]
@ziadoz any progress on this one ? /cc @dunglas

**Comment by theofidry** [comment]
:+1: for this PR

**Comment by dupuchba** [comment]
@ziadoz , can you share insights on this one please ? :-)

**Comment by dupuchba** [comment]
ping @ziadoz :-)

**Comment by dupuchba** [comment]
hello @ziadoz , keep me in touch on this PR status and tell me if I should close it :)

File: README.md [hunk 2]
```diff
@@ -593,6 +593,7 @@ Libraries to help manage database schemas and migrations.
 * [Drest](https://github.com/leedavis81/drest) - A library for exposing Doctrine entities as REST resource endpoints.
 * [Restler](https://github.com/Luracast/Restler) - A lightweight framework to expose PHP methods as RESTful web API.
 * [wsdl2phpgenerator](https://github.com/wsdl2phpgenerator/wsdl2phpgenerator) - A tool to generate PHP classes from SOAP WSDL files.
+* [API Platform](https://api-platform.com ) - Expose in minutes an hypermedia REST API that works out of the box that embraces JSON-LD, Hydra and provides a ton of features (also usable as standalone Symfony bundle).
```
Comment (jadb) [inline README.md]: Can this be shortened a little please? Official description on github repo is shorter (no marketing buzz). :wink:

**Pull request merged**